packages/components/inputs/localized-text-input/src/localized-text-input.tsx

```diff
@@ -149,6 +247,20 @@ const LocalizedTextInput = (props) => {
   const shouldRenderLanguagesButton =
     languages.length > 1 && !props.hideLanguageExpansionControls;
 
+  if (!props.isReadOnly) {
+    warning(
+      typeof props.onChange === 'function',
+      'LocaliszedTextInput: "onChange" is required when isReadOnly is not true'
+    );
+  }
+
+  if (props.hideLanguageExpansionControls) {
+    warning(
+      typeof props.hideLanguageExpansionControls === 'boolean',
+      'LocaliszedTextInput: "defaultExpandLanguages" does not have any effect when "hideLanguageExpansionControls" is set.'
+    );
+  }
```

Comment:
I don't think this is correct. 
I think it's supposed to be something like this;

```suggestion
  if (props.hideLanguageExpansionControls) {
    warning(
      typeof props.defaultExpandLanguages !== 'boolean',
      'LocaliszedTextInput: "defaultExpandLanguages" does not have any effect when "hideLanguageExpansionControls" is set.'
    );
  }
```

Please confirm this.

Reply:
I am not so sure because this was my initial implementation but it breaks a test and I assume we are not meant to change any tests that already satisfies a criteria right?